contentgrid-appserver-query-engine-impl-jooq/src/test/java/com/contentgrid/appserver/query/engine/jooq/JOOQThunkExpressionVisitorTest.java

```diff
@@ -291,27 +307,32 @@ void setup() {
         insertData();
     }
 
+    @Allow.PlainSQL
     void insertData() {
         var now = Instant.now();
         dslContext.insertInto(DSL.table(DSL.name("person")))
                 .set(DSL.field(DSL.name("id"), UUID.class), ALICE_ID)
                 .set(DSL.field(DSL.name("name"), String.class), "alice")
                 .set(DSL.field(DSL.name("vat"), String.class), "vat_1")
+                .set(DSL.field("comment", String.class), "Comment with the words foo and bar.")
```

Comment:
Please update the other statements as well, so you don't need `Allow.PlainSQL`
```suggestion
                .set(DSL.field(DSL.name("comment"), String.class), "Comment with the words foo and bar.")
```